src/model/custom_electra/electra.py

```diff
@@ -0,0 +1,114 @@
+from typing import Optional
+import torch
+import transformers
+from transformers import ElectraForMaskedLM, ElectraForPreTraining 
+from transformers import AutoConfig, PretrainedConfig, PreTrainedModel, AutoModelForMaskedLM
+#from transformers.tokenization_utils import PretrainedTokenizerFast
+from transformers.models.electra.modeling_electra import ElectraForPreTrainingOutput
+
+
+
+class CustomElectraConfig(PretrainedConfig):
+    model_type= "custom_electra"
+
+    def __init__(
+        self,
+        generator_model: transformers.DebertaV2Config,
```

Comment:
(1) rename to `generator_config` (and same for `discriminator`) below.  (2) Just to be fully clear, we don't want to just replace `ElectraX` with `DebertaX`, but rather the most general possible thing that consumes both.  For configs, I honestly think that's just `PretrainedConfig` here